docs/development-guide.md

```diff
@@ -34,6 +34,9 @@ it is **highly** encouraged to:
 * Fork the [ceph-csi repo](https://github.com/ceph/ceph-csi) on Github.
 * Add your fork as a git remote:
    `$ git remote add fork https://github.com/<your-github-username>/ceph-csi`
+* Set up a pre-commit hook to catch issues locally.
+   `$ pip install pre-commit==2.5.1`
+   `$ pre-commit install`
```

Comment:
Are there better ways to install this? `pip` will not automatically install updates when OS updates get installed (with `dnf` or other standard package managers).

Reply:
I followed the installation procedure from  https://pre-commit.com/

Reply:
If we need any other installation procedures, then we can try it out too.

Reply:
I have added the link for other installation instructions in the doc.